mix.exs

```diff
@@ -36,8 +36,10 @@ defmodule ExMachina.Mixfile do
     [
       {:ex_doc, "~> 0.14", only: :dev},
       {:earmark, ">= 0.0.0", only: :dev},
-      {:ecto, "~> 2.1", optional: true},
-      {:postgrex, ">= 0.0.0", only: [:test]},
+      {:ecto, "~> 2.2 or ~> 3.0", optional: true},
+      {:jason, "~> 1.0", optional: true},
+      {:ecto_sql, "~> 2.2 or ~> 3.0", only: :test},
+      {:postgrex, "~> 0.14.0", only: :test},
```

Comment:
Do we need to update `postgrex` or is it fine to leave it as `>= 0.0.0`?

Reply:
`>= 0.0.0` should work fine, `ecto_sql` will already lock it to whatever version it needs.

https://github.com/elixir-ecto/ecto_sql/blob/a11efc4e4d3a6a9025ca96894cb9f708a121d0fd/mix.exs#L52